### DISCUSSION TITLE
Maksymalna długość Hasła MQTT to tylko 29 znaków. Czy można to jakoś przeskoczyć? (mam hasło zdecydowanie dłuższe)

### DISCUSSION THREAD

**Discussion by arek1194** [softwarecrash/EPEver2MQTT#37] [Q&A] [answered]
Hello, My Home Assistant has long ago generated a password much longer than 29 characters for MQTT (I can't change the password anymore) and in the browser, when pasting (or typing) the mqtt password, you can enter a maximum of 29 characters.  Is there any possibility to deal with this?  (I am not proficient in programming) Thank you in advance for any help.

**Comment by softwarecrash** [comment]
english?

**Reply by arek1194** [reply]
Corrected 

"Hello, My Home Assistant has long ago generated a password much longer than 29 characters for MQTT (I can't change the password anymore) and in the browser, when pasting (or typing) the mqtt password, you can enter a maximum of 29 characters.  Is there any possibility to deal with this?  (I am not proficient in programming) Thank you in advance for any help."

**Reply by softwarecrash** [reply, marked as answer]
so the pasword field allow 35 characters, the memory space is 40 char with null termination.

in your case i dont know or i think the chars of your language need some extra space.

so i think 35 characters is enugh for the most people, and i dont modify it, so when i change anything in the memory system it force alle user to re-enter all of the data again.

but you can self compile it for your requirements, its only two lines that need a change

**Reply by arek1194** [reply]
I made a fuss unnecessarily, it turned out that I was entering the wrong password (the correct one was shorter and the field was completely sufficient) Thanks.
